diff --git a/win32/build/config.w32 b/win32/build/config.w32
index ac74085..fe55454 100644
--- a/win32/build/config.w32
+++ b/win32/build/config.w32
@@ -182,7 +182,7 @@ if (VCVERS >= 1500 && PHP_MP != 'disable') {
 
 // General link flags
 DEFINE("LDFLAGS", "/nologo /version:" +
-	PHP_VERSION + "." + PHP_MINOR_VERSION + "." + PHP_RELEASE_VERSION);
+	PHP_VERSION + "." + PHP_MINOR_VERSION);
 
 // General DLL link flags
 DEFINE("DLL_LDFLAGS", "/dll ");
diff --git a/win32/glob.c b/win32/glob.c
index 1aeac78..8111dab 100644
--- a/win32/glob.c
+++ b/win32/glob.c
@@ -61,9 +61,16 @@
  *	Number of matches in the current invocation of glob.
  */
 #ifdef PHP_WIN32
-#define _POSIX_
-#include <limits.h>
-#undef _POSIX_
+#if _MSC_VER < 1800
+# define _POSIX_
+# include <limits.h>
+# undef _POSIX_
+#else
+/* Visual Studio 2013 removed all the _POSIX_ defines, but we depend on some */
+# ifndef ARG_MAX
+#  define ARG_MAX 14500
+# endif
+#endif
 #ifndef S_ISDIR
 #define S_ISDIR(m) (((m) & _S_IFDIR) == _S_IFDIR)
 #endif
